### ISSUE TITLE
How to use without Leaflet?

### ISSUE THREAD

**Issue by rwwagner90** [smeijer/leaflet-geosearch#96]
I am trying to include `dist/bundle.js` and I am getting `Uncaught ReferenceError: L is not defined`. What should I do?

**Comment by smeijer** [comment]
Does that also happen when including from https://unpkg.com? 

```
https://unpkg.com/leaflet-geosearch@latest/dist/bundle.min.js
```

**Comment by smeijer** [comment]
It does indeed also happen with the unpkg version. I'm working on a fix as we speak.

**Comment by smeijer** [comment]
Fixed in `2.0.3` (just published). See working example here: http://codepen.io/smeijer/pen/VPVqjZ?editors=1011
